gossip/src/gossip_service.rs

```diff
@@ -195,7 +195,7 @@ pub fn get_clients(nodes: &[ContactInfo], socket_addr_space: &SocketAddrSpace) -
     nodes
         .iter()
         .filter_map(|node| ContactInfo::valid_client_facing_addr(node, socket_addr_space))
-        .map(create_client)
+        .map(|addrs| create_client(addrs.0, addrs.1))
```

Comment:
prefer destructuring (throughout)
```suggestion
        .map(|(rpc, tpu)| create_client(rpc, tpu))
```